Geometry/HGCalGeometry/src/HGCalGeometry.cc

```diff
@@ -125,527 +107,478 @@ void HGCalGeometry::newCell( const GlobalPoint& f1 ,
+    co[ncorner - 1] = GlobalPoint(0, 0, -2 * dz);
   }
   return co;
 }
 
-DetId HGCalGeometry::neighborZ(const DetId& idin, 
-			       const GlobalVector& momentum) const {
+DetId HGCalGeometry::neighborZ(const DetId& idin, const GlobalVector& momentum) const {
   DetId idnew;
   HGCalTopology::DecodedDetId id = m_topology.decode(idin);
-  int lay = ((momentum.z()*id.zSide > 0) ? (id.iLay+1) : (id.iLay-1));
+  int lay = ((momentum.z() * id.zSide > 0) ? (id.iLay + 1) : (id.iLay - 1));
 #ifdef EDM_ML_DEBUG
-  edm::LogVerbatim("HGCalGeom") << "neighborz1:: ID " << id.iLay << ":"
-				<< id.iSec1 << ":" << id.iSec2 << ":"
-				<< id.iCell1 << ":" << id.iCell2 
-				<< " New Layer " << lay << " Range "
-				<< m_topology.dddConstants().firstLayer() <<":"
-				<< m_topology.dddConstants().lastLayer(true)
-				<< " pz " << momentum.z();
+  edm::LogVerbatim("HGCalGeom") << "neighborz1:: ID " << id.iLay << ":" << id.iSec1 << ":" << id.iSec2 << ":"
+                                << id.iCell1 << ":" << id.iCell2 << " New Layer " << lay << " Range "
+                                << m_topology.dddConstants().firstLayer() << ":"
+                                << m_topology.dddConstants().lastLayer(true) << " pz " << momentum.z();
 #endif
-  if ((lay >= m_topology.dddConstants().firstLayer()) && 
-      (lay <= m_topology.dddConstants().lastLayer(true)) && 
+  if ((lay >= m_topology.dddConstants().firstLayer()) && (lay <= m_topology.dddConstants().lastLayer(true)) &&
       (momentum.z() != 0.0)) {
     GlobalPoint v = getPosition(idin);
-    double      z = id.zSide*m_topology.dddConstants().waferZ(lay,true); 
-    double   grad = (z - v.z())/momentum.z();
-    GlobalPoint p(v.x()+grad*momentum.x(),v.y()+grad*momentum.y(),z);
-    double      r = p.perp();
-    auto rlimit   = topology().dddConstants().rangeR(z,true);
-    if (r >= rlimit.first && r <= rlimit.second) idnew = getClosestCell(p);
+    double z = id.zSide * m_topology.dddConstants().waferZ(lay, true);
+    double grad = (z - v.z()) / momentum.z();
+    GlobalPoint p(v.x() + grad * momentum.x(), v.y() + grad * momentum.y(), z);
+    double r = p.perp();
+    auto rlimit = topology().dddConstants().rangeR(z, true);
+    if (r >= rlimit.first && r <= rlimit.second)
+      idnew = getClosestCell(p);
 #ifdef EDM_ML_DEBUG
-  edm::LogVerbatim("HGCalGeom") << "neighborz1:: Position " << v << " New Z "
-				<< z << ":" << grad << " new position " << p
-				<< " r-limit " << rlimit.first << ":" 
-				<< rlimit.second;
+    edm::LogVerbatim("HGCalGeom") << "neighborz1:: Position " << v << " New Z " << z << ":" << grad << " new position "
+                                  << p << " r-limit " << rlimit.first << ":" << rlimit.second;
 #endif
   }
   return idnew;
 }
 
-DetId HGCalGeometry::neighborZ(const DetId& idin, 
-			       const MagneticField* bField, int charge,
-			       const GlobalVector& momentum) const {
+DetId HGCalGeometry::neighborZ(const DetId& idin,
```

Comment:
how do you decide which function parameters should be all in one line or each in separate one?